src/include/liburing/io_uring.h

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR MIT */
+/* SPDX-License-Identifier: (GPL-2.0-only WITH Linux-syscall-note) OR MIT */
```

Comment:
I'd rather not make this change unless I absolutely have to, since the header is synced with the kernel and then I have to make that change there too.

Reply:
Ah ok, makes sense, I guess this would need to be fixed in the kernel then, but that might imply lots of instances being touched. I'll pull this one out then.